presto-main/src/main/java/com/facebook/presto/statusdetector/NodeStatusDetector.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.statusdetector;
+
+import java.util.List;
+
+public interface NodeStatusDetector
```

Comment:
just thinking about the name `NodeStatusDetector`, I think detector feels like it detects when the node is up/down/disabled etc. This is more of a `NodeStatusService` which provides a list of disabled servers.

Reply:
done.